u2-routing-controllers/features/rgbController.js

```diff
@@ -0,0 +1,27 @@
+
+const getrgb = (req, res) => {
+  const r = parseInt(req.query.r) || 0
+  const g = parseInt(req.query.g) || 0
+  const b = parseInt(req.query.b) || 0
+  const avg = Math.round((r + g + b) / 3)
+
+  if (avg < 128) {
+    res.send(`
+    <html>
+      <body style="background-color: rgb(${r}, ${g}, ${b}) ;">
+       <font color=white>
+        red: ${r}, green: ${g}, blue: ${b} <br>
+        avg: ${avg} 
+       </font>
```

Comment:
ฉันแนะนำให้คุณใช้ `style="background-color: ...; color: #fff"`
`<font>` เลิกใช้แล้ว (& CSS นั้นสะอาดกว่า)

I encourage you to use `style="background-color: ...; color: #fff"`
`<font>` is deprecated (& CSS is cleaner)